Automattic/woocommerce-payments
# Add card readers page business url field client validation

Fixes #3952

#### Changes proposed in this Pull Request
Adding a client-side validation for the Business URL setting field on the Card Readers settings page.

<img width="552" alt="Screenshot 2022-03-21 at 17 55 57" src="https://user-images.githubusercontent.com/8667118/159300204-f0c1512b-900f-4139-86b4-c5522d155546.png">

#### Testing instructions
- Update the `_wcpay_feature_card_readers` option to `1`.
- Update the `card_present_eligible` to `true`.
- Navigate to `Payments` -> `Card Readers page` -> `Receipt details`.
- Enter invalid URL address in Business URL field. 
- Verify error is displayed and Save button is disabled.

-------------------
